README.md

```diff
@@ -275,6 +280,176 @@ Reads if device geolocation is activated or not (read-only).
+| `update.zipcode` | `string[]` | Record postal/zip code(s). |
+| `update.population` | `number?` | Record population of its city. |
+| `origin` | `key-value mapping` (optional) | Original (and complete) record to fix, from `pk.search()` or `pk.reverse()`. |
+| `opts` | `key-value mapping` (optional) | Patch options. |
+| `opts.status` | `('pending' | 'approved')?` | Publication status. |
+| `opts.language` | `string?` | Target language, [two-letter ISO](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code. |
+
+### `pk.patch.retrieve()`
+
+Retrieve a patch record by ID.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+pk.patch.retrieve('<patch-id>').then((record) => {
+  console.log(record);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `id` | `string` | Record ID. |
+
+### `pk.patch.update()`
+
+Update a patch record.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// update and publish
+pk.patch.update(
+  '<patch-id>',
+  { coordinates: '33.9955095,-118.472482' },
+  { status: 'approved' }
+).then((record) => {
+  console.log(record);
+});
+
+// update translation
+pk.patch.update(
+  '<patch-id>',
+  { name: 'Rue exemple' }, 
+  { language: 'fr' }
+).then((record) => {
+  console.log(record);
+});
+
+// unpublish
+pk.patch.update(
+  '<patch-id>',
+  undefined,
+  { status: "pending" }
+).then((record) => {
+  console.log(record);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `id` | `string` | Record ID. |
+| `update` | `key-value mapping` (optional) | Updated fields. |
+| `update.type` | `string` | One of `airport`, `bus`, `city`, `street`, `tourism`, `townhall`, `train`. |
+| `update.name` | `string` | Record display name (street name, city name, station name...). |
+| `update.city` | `string` | Record city name. |
+| `update.county` | `string` | Record county/province/department. |
+| `update.administrative` | `string` | Record administrative/region/state. |
+| `update.country` | `string` | Record country name. |
+| `update.countrycode` | `string` | Record [two-letter ISO](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. |
+| `update.coordinates` | `string` | Record coordinates in format `lat,lng`. |
+| `update.zipcode` | `string[]` | Record postal/zip code(s). |
+| `update.population` | `number?` | Record population of its city. |
+| `opts` | `key-value mapping` (optional) | Patch options. |
+| `opts.status` | `('pending' | 'approved')?` | Publication status. |
+| `opts.language` | `string?` | Target language, [two-letter ISO](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code. |
+
+### `pk.patch.delete()`
+
+Delete a patch record or a patch translation.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// delete patch translation
+await pk.patch.delete('<patch-id>', 'fr');
```

Comment:
Seems like a bad idea afterall :D You better not forget to put the translation, because you will end up deleting your record. Not very dev UX

Reply:
Kinda bummer to add a whole method just for this :(. But agree, could be prone to mistakes.